include/revng/PipelineC/Prototypes.h

```diff
@@ -408,6 +475,36 @@ bool rp_container_store(rp_container *container, const char *path);
  */
 bool rp_container_load(rp_container *container, const char *path);
 
+/**
+ * \return the serialized content of the element associated to the provided
+ * target,
+ *
+ * \note Target must be already present in container
+ *
+ */
+const char *
+rp_container_extract_one(rp_container *container, rp_target *target);
+
+/** \} */
+
+/**
+ *
+ * \defgroup rp_diff_map rp_diff_map methods
+ * \{
+ */
+
+/**
+ * frees the provided map
+ */
+void rp_diff_map_destroy(rp_diff_map *to_free);
+
+/**
+ * \returns nullptr if global_name did not named a global variable in the
+ * diff_map else return the serialized diff of the indicated global
+ */
+const char * /*owning*/
+rp_diff_map_get_diff(rp_diff_map *map, const char *global_name);
```

Comment:
Argh, I suspect multi-line declaration might break `grep owning` `:(`
What can we do about it?

Reply:
jack said he handles it

Reply:
Yes, multiline is not a problem